### PR TITLE
Pixel android 14 new packages

### PR DESCRIPTION
**Pull request by Blitz517**

### CODE REVIEW

**Review by AnonymousWP — CHANGES_REQUESTED** [review]
See comments and suggestions.

File: resources/assets/uad_lists.json [hunk 1]
```diff
@@ -37511,6 +37517,207 @@
     "id": "com.motorola.android.fota",
     "list": "Carrier",
     "description": "AT&T Software Update, But by the carrier.",
+    "id": "com.google.android.carrierlocation",
+    "list": "Oem",
+    "description": "Sharing location to carrier.",
```
[the remark this answers is not in the document]
Comment (Blitz517) [reply resources/assets/uad_lists.json:37522]: hm its carrier location, not any emergency stuff, but resources.arsc cant open normally, i found something about this: https://support.google.com/pixelphone?p=carrier_location_permissions
maybe it should be ``expert`` set

File: resources/assets/uad_lists.json [hunk 2]
```diff
@@ -37511,6 +37517,207 @@
...
+    "description": "Has too much random logging, metrics stuff.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.google.android.hardwareinfo",
+    "list": "Oem",
+    "description": "Needed for VoLTE? Has a lot statistics and takes info about device.",
```
[the remark this answers is not in the document]
Comment (Blitz517) [reply resources/assets/uad_lists.json:37621]: this app requires gms, has call notification but maybe theres no stuff about volte, maybe remove this ``Needed for VoLTE?`` this app is on some websites on the list degoogling.

[the remark this answers is not in the document]
Comment (AnonymousWP) [reply resources/assets/uad_lists.json:37621]: Yes sure, go ahead, but should this still be `Recommended` then? I don't think so, as we're still not sure about its implications.

**Comment by AnonymousWP** [comment]
Superseded & resolved by https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/pull/273.

**Pull request closed without merging**